lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2600,8 +2627,14 @@ GLStatus TFGraphLowering::lowerConditionalRegion(ConditionalSESERegion *r) {
 
   // Remember each of the results so that any references to the SIL BBArguments
   // that got defined end up referring to this node.
-  for (int i = 0, e = trueCodeFn.outputs.size(); i != e; ++i)
-    addValueMapping({trueCodeFn.outputs[i].first, 0}, {result, i});
+  graphFn.tentative_outputs.clear();
+  for (int i = 0, e = trueCodeFn.outputs.size(); i != e; ++i) {
```

Comment:
Why not unsigned?

Reply:
the c'tor of TF_Output() requires a signed int